core/jobs/base_jobs.py

```diff
@@ -75,6 +75,9 @@ class JobMetaclass(type):
 
     _JOB_REGISTRY: Dict[str, Type[JobBase]] = {}
 
+    # Here we use type Any because argument 'namespace' can accept methods,
```

Comment:
I'm not sure about this. In practice, what are the types of values that are actually used? We can specialize to those.

Reply:
I didn't find any specific case -- which explains the types that this argument can accept. For now, I have just put str here and it worked. However, in the future, if we encounter an error then we can add Union here.